src/RestApi/Controllers/ProductCategories.php

```diff
@@ -117,6 +117,27 @@ public function prepare_item_for_response( $item, $request ) {
 			'permalink'   => get_term_link( $item->term_id, 'product_cat' ),
 		);
 
+		if ( $request->get_param( 'show_review_count' ) ) {
+			global $wpdb;
+			$term_id       = $item->term_id;
+			$term_taxonomy = 'product_cat';
+
+			$products_of_category_sql = "
+			SELECT SUM( DISTINCT comment_count) as review_count
+			FROM {$wpdb->posts} AS posts
+			INNER JOIN {$wpdb->term_relationships} AS term_relationships ON posts.ID = term_relationships.object_id
+			INNER JOIN {$wpdb->term_taxonomy} AS term_taxonomy USING( term_taxonomy_id )
+			INNER JOIN {$wpdb->terms} AS terms USING( term_id )
+			WHERE terms.term_id IN ( {$term_id} )
+			AND term_taxonomy.taxonomy IN ( '{$term_taxonomy}' )
+			";
+			$products_of_category = $wpdb->get_results($products_of_category_sql); // phpcs:ignore
```

Comment:
If you use `wpdb->get_var` instead of `wpdb->get_results`, you'll get a single value instead of an array of values.